### ISSUE TITLE
paf1 complex, check processes

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#2177]
Cdc73/Paf1 complex
--
  | cdc73, leo1, paf1, prf1, tpr1

https://www.ncbi.nlm.nih.gov/pubmed/28585565 Paf1C function in transcription through chromatin.

**Comment by ValWood** [comment]
The Paf1 complex factors Leo1 and Paf1 promote local histone turnover to modulate chromatin states in fission yeast.
Sadeghi L1, Prasad P1, Ekwall K2, Cohen A3, Svensson JP2.
PMID: 26518661 


Seems to emphasise the role in histone trunover.
However, this is somehow related to maintaining chromatin states for active transcription which isn't captured

**Comment by ValWood** [comment]
In addition to the intrinsic function of Paf1 and Leo1 in promoting RNAPII‐mediated chromatin transcription 40, these components of the Paf1 complex are involved in multiple activities that facilitate transcription elongation. In budding yeast, Paf1C interacts with Set1 to methylate Lys4 on histone H3 and with the ubiquitin ligase Bre1 to monoubiquitinate Lys120 on histone H2B 42, 43. In S. pombe, Set1‐mediated H3 methylation is conserved, and the corresponding H2B residue Lys119 is monoubiquitinated by the Bre1 homologs Brl1 and Brl2 44, 45. We confirmed a previous observation that H2Bub1 was partially dependent on Leo1 because H2Bub1 was reduced in leo1Δ (61% of WT levels), albeit not to the same extent as the H2Bub1‐deficient htb1‐K119R (Fig 2D).

To determine whether paf1Δ and leo1Δ indirectly caused heterochromatin propagation across IR‐L by impairing H2Bub1 or H3K4me, we examined cells with deficient H2B ubiquitination (rfp1Δ, shf1Δ, and htb1‐K119R) or H3K4 methylation (set1Δ) with regard to chromatin repression (Fig 2E and F). In contrast to paf1Δ or leo1::Hermes, the mutations negatively affecting H2Bub1 or H3K4me did not induce gene silencing across the heterochromatin boundary at IR‐L. This result is in agreement with a functional separation between the Leo1–Paf1 heterodimer and the rest of the Paf1 complex containing Rfp1 39. Together, these data indicate that although Paf1C is involved in recruiting factors to generate H2Bub1 and H3K4me, the role of the Leo1–Paf1 subcomplex in heterochromatin spreading across the IR‐L boundary is distinct from the functions of its known interactors. Thus, heterochromatin propagation is not related to the transcriptional marks H2Bub1 and H3K4me.

(is this the complex wghich allows txn through faculcative heterochromatin?, i.e enables elongation through difficult regions?
